examples/nft-nd-nns/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.22
 
 require (
 	github.com/nspcc-dev/neo-go v0.106.4-0.20241016130346-d8e945978af6
-	github.com/nspcc-dev/neo-go/pkg/interop v0.0.0-20241212130705-ea0a6114d2d6
+	github.com/nspcc-dev/neo-go/pkg/interop v0.0.0-20241223113753-f3e22b707014
```

Comment:
Good, and then in a separate commit you need to update `github.com/nspcc-dev/neo-go` dependency.